presto-hive/src/test/java/com/facebook/presto/hive/s3/TestPrestoS3FileSystem.java

```diff
@@ -145,15 +146,16 @@ public void testInstanceCredentialsEnabled()
         }
     }
 
-    @Test(expectedExceptions = RuntimeException.class, expectedExceptionsMessageRegExp = "S3 credentials not configured")
-    public void testInstanceCredentialsDisabled()
+    @Test
+    public void testDefaultCredentials()
             throws Exception
     {
         Configuration config = new Configuration();
         config.setBoolean(S3_USE_INSTANCE_CREDENTIALS, false);
 
         try (PrestoS3FileSystem fs = new PrestoS3FileSystem()) {
             fs.initialize(new URI("s3n://test-bucket/"), config);
+            assertInstanceOf(getAwsCredentialsProvider(fs), DefaultAWSCredentialsProviderChain.class);
```

Comment:
seems like you could an 
`assertEquals(getAwsCredentialsProvider(fs), DefaultAWSCredentialsProviderChain.getInstance())`
which is better than `assertInstanceOf`

Reply:
I recommend keeping this, since this is more descriptive and the `DefaultAWSCredentialsProviderChain ` class is a stronger contract than the instance.